apps/web/.env

```diff
@@ -1,18 +1,3 @@
 # Uncomment the next line to test a local API / Auth
 #NEXT_PUBLIC_ENV=local
-NEXT_PUBLIC_ENV=staging
-NEXT_PUBLIC_SERLO_DOMAIN_PRODUCTION=serlo.org
-NEXT_PUBLIC_SERLO_DOMAIN_STAGING=serlo-staging.dev
-NEXT_PUBLIC_SERLO_DOMAIN_LOCAL=serlo.localhost:4567
-# empty string if not on vercel
-NEXT_PUBLIC_VERCEL_URL=
-# empty string if not on vercel
-NEXT_PUBLIC_COMMIT_SHA=
-HYDRA_CLIENT_ID_STAGING=frontend
-HYDRA_CLIENT_SECRET_STAGING=YjgwZThhZDk0MTYxMTA4ODY1NzYxZmJk
-HYDRA_HOST_STAGING=https://hydra.${NEXT_PUBLIC_SERLO_DOMAIN_STAGING}
-HYDRA_CLIENT_ID_LOCAL=frontend.serlo.org
-HYDRA_CLIENT_SECRET_LOCAL=frontend.serlo.org
-HYDRA_HOST_LOCAL=http://localhost:4444
-KRATOS_HOST_STAGING=https://kratos.${NEXT_PUBLIC_SERLO_DOMAIN_STAGING}
-KRATOS_HOST_LOCAL=http://localhost:4433
\ No newline at end of file
+NEXT_PUBLIC_ENV=staging
```

Comment:
just an idea: If we would add a script like this we could remove `.env` as well and make it even easier to find this functionality.
`"dev-local": "cross-env NEXT_PUBLIC_ENV=local next",`

Reply:
I guess the downside is that we either have to fall back to staging sometimes (like we already do in the code below) or set `cross-env NEXT_PUBLIC_ENV=staging ` for the `dev` script as well.
```
// if env is not set, it's a production build running on localhost, use staging as default
const KRATOS_HOST =
  KRATOS_HOSTS[
    process.env.NEXT_PUBLIC_ENV ? process.env.NEXT_PUBLIC_ENV : 'staging'
  ]
```